tutorials/00_Git_and_GitHub_workflow.md

```diff
@@ -49,14 +49,54 @@ upstream repository (the original remindmodel fork).
 
 ## 2.2 Clone (download your personal repository)
 
-It is basically possible to change the code just using the GitHub
-interface, but since you want to test and run your code locally you have
-to clone the repository from your fork anyway. This can be done using
-the 'https' or 'ssh' address of your fork together with the `git clone`
-command (cmd/bash/GUI) at your machine. We recommend to upload an
-ssh-key and use ssh to connect to GitHub. Under Windows open a git bash as follows: 
-open the file explorer, right-click on the folder you want to execute the git commands in
-and choose "Git Bash Here" from the context menu. You need to have git installed.
+To run REMIND on your local machine or a high-performance cluster, you have to clone (download) the code from your fork.
```

Comment:
People actually do that and pull `develop` from their fork?  Weird.

Reply:
Yeah, it means you have to either
* update your fork periodically on the github page of your fork, then `git pull` from your fork
* `git pull upstream develop` and `git push origin` periodically to update the develop branch in your fork.

That's a bit weird. On the other hand, people really like that they have a copy of everything (even the "develop" branch) and can break it as they wish. Many work directly on this develop branch, and pull from upstream only if they need to. It is a workflow which makes sure you don't accidentally push to remindmodel/remind develop (not that you could, given then branch protection rules, but anyway, belts and suspenders), and it makes all the steps very explicit.